runzero-api.yml

```diff
@@ -1249,6 +1249,184 @@ paths:
         '404':
           $ref: '#/components/responses/NotFoundError'
 
+  /org/explorers:
+    parameters:
+      - $ref: '#/components/parameters/orgID'
+    get:
+      tags:
+        - Organization
+      operationId: getExplorers
+      summary: Get all explorers
+      description: Get all explorers. This is the same call as legacy path, /org/agents
```

Comment:
I don't think the commas belong between "legacy path" and the path.

Reply:
I think, you may be quite, correct, @lpar. Fixed!